code/AssetLib/glTF/glTFImporter.cpp

```diff
@@ -664,7 +664,7 @@ void glTFImporter::ImportEmbeddedTextures(glTF::Asset &r) {
 
                 const size_t len = strlen(ext);
                 if (len <= 3) {
-                    strncpy(tex->achFormatHint, ext, len);
+                    strncpy(tex->achFormatHint, ext, len+1);//+1 because we need to copy terminator
```

Comment:
This will cause an override when the length is more than 4 bytes. Just using a fixed-sized array would work better. The default size is 4.

Reply:
isn't the check len<=3 on top guarantees it won't be bigger than 4? That was what I though at least.